docs/develop/node/maintenance/maintenance.md

```diff
@@ -42,9 +30,13 @@ Once `MainNet: Restricted` is live, planned updates to `TestNet` and `MainNet` w
 
 ## Emergency Updates
 
+We may issue a `[CODE_YELLOW_BETANET]` or `[CODE_YELLOW_TESTNET]` if the network is suffering minor issues, or a new software release introduces incompatibilities and requires additional testing.
```

Comment:
Don't think we do this for betanet. Probably better to use mainnet as an example